### DISCUSSION TITLE
🍭反馈建议

### DISCUSSION THREAD

**Discussion by perfree** [perfree/PerfreeBlog#16] [Ideas]
🍭如果您有什么对于该项目好的想法和建议, 可以在此提交下~🍻🍻🍻

**Comment by 111de000** [comment]
这个系统的插件怎么使用呢。还有为什么本地部署的网页点击跳转404来着

**Reply by perfree** [reply]
插件使用可参考官网文档：https://perfree.org.cn/useDocs/plugins.html
 本地开发为前后端分离项目， 需要单独访问前端端口
